src/jdk.jpackage/share/classes/jdk/jpackage/main/Main.java

```diff
@@ -45,10 +47,25 @@
      *
      * @param args command line arguments
      */
+    private final static Charset nativeCharset;
```

Comment:
Static initializer dissecting main method (javadoc/impl)

Reply:
Moved to another place